ros/src/waypoint_updater/waypoint_updater.py

```diff
@@ -51,54 +67,141 @@ def loop(self):
         while not rospy.is_shutdown():
             if self.pose and self.waypoints and self.waypoints_2d and self.waypoint_tree:
```

Comment:
Can you please add `and self.traffic_wp_idx` here? I'd like `waypoint_updater` to avoid publishing anything until it receives the first classification of light.

Reply:
`and self.traffic_wp_idx is not None` would be better, just in case there is a traffic light at wp 0